tutorial/developing_packages/local_package_development_flow/conanfile.py

```diff
@@ -0,0 +1,54 @@
+import os
```

Comment:
I wonder if the development flow tutorial should be optimized for the in-source recipe case, which is the most expected for users doing local development, and not for third-party source download flow?